war/gulpfile.js

```diff
@@ -23,7 +23,7 @@ builder.bundle('src/main/js/pluginSetupWizard.js')
     .withExternalModuleMapping('bootstrap', 'core-assets/bootstrap:bootstrap3', {addDefaultCSS: true})
     .withExternalModuleMapping('handlebars', 'core-assets/handlebars:handlebars3')
     .less('src/main/less/pluginSetupWizard.less')
-    .inDir('src/main/webapp/jsbundles');
+    .inDir('target/classes/org/jenkinsci/setup'); // bundle as an adjunct
```

Comment:
@tfennelly I agree but then we should replace all explicit references that assume these resources are available under `[context]/jsbundles/`. And there is other way to do the same (or similar), here an [example](https://github.com/recena/jenkins/blob/JENKINS-34244/core/src/main/resources/hudson/model/View/newJob.jelly#L31-32).

What is the righy way?